oauthproxy_test.go

```diff
@@ -80,7 +81,7 @@ func TestWebSocketProxy(t *testing.T) {
 
 	backendURL, _ := url.Parse(backend.URL)
 
-	options := options.NewOptions()
+	options := baseTestOptions()
```

Comment:
I've started renaming these to `opts` whenever I saw them in tests as well.  I think `options` shadows the same name `options` import.